### PR TITLE
update docs

### PR DESCRIPTION
**Pull request by v-zhongz**

### CODE REVIEW

File: docs/template-guide.md [hunk 1]
```diff
@@ -290,7 +314,7 @@ Sample output:
 	Deployment
  	not set
 
-2)	 You can reference the example file [cf_212.yml](http://cloudfoundry.blob.core.windows.net/misc/cf_212.yml) to replace the **BOSH-DIRECTOR-UUID**, **VNET-NAME**, **SUBNET-NAME**, **RESERVED-IP** and **SSL-CERT-AND-KEY** properties.
+2)	 You can reference the example file (http://cloudfoundry.blob.core.windows.net/misc/cf_212.yml) to replace the **BOSH-DIRECTOR-UUID**, **VNET-NAME**, **SUBNET-NAME**, **RESERVED-IP** and **SSL-CERT-AND-KEY** properties. The BOSH-DIRECTOR-UUID, VNET-NAME and SUBNET-NAME are created in step 2. The RESERVED-IP can be got from the row "cf-ip" in /home/YOUR-USERNAME/settings. To replace the SSL-CERT-AND-KEY, you need to link the cert in /home/YOUR-USERNAME/bosh.yml and the key in /home/YOUR-USERNAME/bosh.
```
Comment (AbelHu) [inline docs/template-guide.md]: "\[ \]\(\)" is markdown file format. If you want to add hyperlink for the example file, you should use "\[the example file\]\(http://cloudfoundry.blob.core.windows.net/misc/cf_212.yml)"

Comment (AbelHu) [inline docs/template-guide.md]: The RESERVED-IP can be got 
got => gotten

To replace the SSL-CERT-AND-KEY, you need to link the cert in /home/YOUR-USERNAME/bosh.yml and the key in /home/YOUR-USERNAME/bosh.
What is the link?

Comment (v-zhongz) [inline docs/template-guide.md]: @AbelHu Revised.

**Comment by bingosummer** [comment]
@v-zhongz You can rebase your commits into one clean commit. Please add more readable comments instead of "Update doc".

**Comment by v-zhongz** [comment]
@bingosummer Changes are fragmentary here. Checking "Files changed" is clearer.

**Pull request closed without merging**